packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1350,6 +1350,7 @@ const messages = {
       h5p: "H5P",
       external: "Ekstern ressurs",
       gloss: "Glose",
+      copyright: "Tekst",
```

Comment:
Er dette for at det skal dukke opp i Tekst-fanen?

Reply:
Nei. Dette er teksten som står i byline.